### PR TITLE
[GCE] Ingress HTTP2 e2e test

### PR DESCRIPTION
**Pull request by agau4779**
**What this PR does / why we need it**:
- Adds e2e test for bringing up an HTTP2 Ingress, converting it to HTTPS, then back to HTTP2
- Update echoserver image to 1.10

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: test/e2e/network/ingress.go [hunk 1]
```diff
@@ -575,6 +575,119 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 		})
 	})
 
+	Describe("GCE [Slow] [Feature:HTTP2]", func() {
```
Comment (freehan) [inline test/e2e/network/ingress.go]: nit: no need a different section. place it under [Feature: Ingress]

File: test/e2e/network/ingress.go [hunk 2]
```diff
@@ -575,6 +575,119 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
...
+		It("should be able to switch between HTTPS and HTTP2 modes", func() {
+			By("Create a basic HTTP2 ingress")
+			jig.CreateIngress(filepath.Join(framework.IngressManifestPath, "http2"), ns, map[string]string{}, map[string]string{})
+			jig.WaitForIngress(true)
+
+			address, err := jig.WaitForIngressAddress(jig.Client, jig.Ingress.Namespace, jig.Ingress.Name, framework.LoadBalancerPollTimeout)
+
+			By(fmt.Sprintf("Polling on address %s and verify the backend is serving HTTP2", address))
+			timeoutClient := &http.Client{Timeout: framework.IngressReqTimeout}
+			err = wait.PollImmediate(framework.LoadBalancerPollInterval, framework.LoadBalancerPollTimeout, func() (bool, error) {
```
Comment (freehan) [inline test/e2e/network/ingress.go]: write a helper function to detect HTTP version. Reuse the helper function in the test.

File: test/e2e/network/ingress.go [hunk 3]
```diff
@@ -382,6 +382,39 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 			// TODO(nikhiljindal): Check the instance group annotation value and verify with a multizone cluster.
 		})
 
+		It("should be able to switch between HTTPS and HTTP2 modes", func() {
+			httpsScheme := "request_scheme=https"
```
Comment (bowei) [inline test/e2e/network/ingress.go]: const

**Pull request merged**